src/pages/AddItem.tsx

```diff
@@ -73,10 +74,11 @@ function AddItem () {
             <Input label="판매가격" name="price" type="number" placeholder="판매 가격을 입력해주세요" required />
             <Input label="태그" name="tag" placeholder="태그를 입력해주세요" onKeyDown={handleKeyDown}>
                 <div className="tags">
-                    {tags.map((element, index) => 
-                        (<Tag key={element} onXClick={() => setTags((prev) => prev.filter((ele, ind) => (index !== ind)))}>
+                    {tags.map((element, index) => (
+                        <Tag key={index} onXClick={() => setTags((prev) => prev.filter((ele, ind) => (index !== ind)))}>
```

Comment:
요구사항에 해당하는 부분은 아니나 질문 남깁니다.
`filter` 메소드의 콜백 함수에 index를 사용하려면 인자를 2개로 받아야 하는 것으로 알고 있습니다. 그래서 2개의 인자를 받고 앞의 인자(element)는 사용하지 않았는데, 이렇게 작성하니 경고가 뜹니다. 콜백 함수에서 index 만 받아오는 방법이 있을까요? 없다면 그냥 `ele`을 반강제로 사용하도록 코드를 작성해야 하나요?

Reply:
우선 사용하지 않는 요소를 선언해서 나오는 에러로 이런경우 아래처럼 작성하시면 IDE에서 미사용인자라는 것을 알고 경고가 나오지 않습니다.

```suggestion
                        <Tag key={index} onXClick={() => setTags((prev) => prev.filter((_, ind) => (index !== ind)))}>
```

다만 index로 filter하시는 것 보다 element 자체로 filter 하시는 것이 더 정확할 것 같아 ele을 사용하시는 것을 추천드리고 싶어요~

Reply:
element(태그 내용)이 동일한 태그가 있을 경우 잘못 제거될 수 있을 것 같아 index로 사용하였습니다. 답변 감사합니다!